cmd/node/config/p2p.toml

```diff
@@ -24,6 +24,12 @@
     #the sync and consensus mechanisms
     ThresholdMinConnectedPeers = 3
 
+    # ConnectionWatcherType represents the type of a connection watcher needed.
+    # possible options:
+    #  - "disabled" - no connection wathing should be made
```

Comment:
[comment]

I know this is not part of your PR, but perhaps rename the typo in `wathing` ?